test/integration/anchorplatform.test.ts

```diff
@@ -0,0 +1,144 @@
+import { Wallet } from "../../src";
+import { IssuedAssetId } from "../../src/walletSdk/Asset";
+
+let wallet;
+let stellar;
+let anchor;
+let accountKp;
+const anchorUrl = "http://localhost:8080";
+
+describe("Anchor Platform Integration Tests", () => {
+  beforeAll(async () => {
+    // Setup
+    wallet = Wallet.TestNet();
+    stellar = wallet.stellar();
+    anchor = wallet.anchor({ homeDomain: anchorUrl, allowHttp: true });
+    accountKp = stellar.account().createKeypair();
+    await stellar.fundTestnetAccount(accountKp.publicKey);
+  }, 15000);
+
+  it("SEP-10 auth should work", async () => {
+    const auth = await anchor.sep10();
+    const authToken = await auth.authenticate({ accountKp });
+    expect(authToken.token).toBeTruthy();
+  });
+
+  it("SEP-12 KYC and SEP-6 should work", async () => {
+    const auth = await anchor.sep10();
+    const authToken = await auth.authenticate({ accountKp });
+
+    // add USDC trustline
+    const asset = new IssuedAssetId(
+      "USDC",
+      // anchor platform USDC issuer
+      "GDQOE23CFSUMSVQK4Y5JHPPYK73VYCNHZHA7ENKCV37P6SUEO6XQBKPP",
+    );
+    const txBuilder = await stellar.transaction({
+      sourceAddress: accountKp,
+    });
+    const addUsdcTx = txBuilder.addAssetSupport(asset).build();
+    addUsdcTx.sign(accountKp.keypair);
+    await stellar.submitTransaction(addUsdcTx);
+
+    // add SEP-12 KYC info
+    const sep12 = await anchor.sep12(authToken);
+    const sep12Resp = await sep12.add({
+      sep9Info: {
+        first_name: "john",
+        last_name: "smith",
+        email_address: "123@gmail.com",
+        bank_number: "12345",
+        bank_account_number: "12345",
+      },
+    });
+    expect(sep12Resp.data.id).toBeTruthy();
+
+    // SEP-6 deposit
+    const sep6 = anchor.sep6();
+    const dResp = await sep6.deposit({
+      authToken,
+      params: {
+        asset_code: "USDC",
+        account: accountKp.publicKey,
+      },
+    });
+    expect(dResp.id).toBeTruthy();
+
+    // SEP-6 withdraw
+    const wResp = await sep6.withdraw({
+      authToken,
+      params: {
+        asset_code: "USDC",
+        account: accountKp.publicKey,
+        type: "bank_account",
+        dest: "123",
+        dest_extra: "12345",
+      },
+    });
+    expect(wResp.id).toBeTruthy();
```

Comment:
Can you also check transaction status?

Reply:
this one actually doesn't have a status, the return body is just:
`{ id: 'd4c34a3d-4f2f-443c-b9be-320d385390b7' }`

Reply:
No, I mean getting transaction and checking status